project/.travis.yml.twig

```diff
@@ -42,6 +42,7 @@ env:
     - SYMFONY_DEPRECATIONS_HELPER=weak_vendors
     - TARGET=test
     - UPSTREAM_URL=https://github.com/sonata-project/{{ repository_name }}.git
+    - PHPUNIT_VERSION={{ '5.6' in php ? '5.7', '7' }}
```

Comment:
I'm mitigated.

It is a nice improvement right now, but can be more complicated later depending to compatibility issues.

Reply:
If it does, we may change in favor of explicit versions in the config, but until then, this is better, isn't it?

Reply:
Yes it is